index.html

```diff
@@ -163,9 +164,15 @@ <h1 id="title"><i class="todo">[DRAFT]</i> Media Working Group Charter</h1>
         </table>
       </div>
 
+      <div id="background" class="background">
+        <h2>Motivation and Background</h2>
+        <p>Standardization efforts to develop media foundations for the Web, such as the <a href="https://html.spec.whatwg.org/#htmlmediaelement"><code>HTMLMediaElement</code></a> interface and <a href="https://www.w3.org/TR/media-source/">Media Source Extensions</a>, have helped turn the Web into a major platform for media streaming and media consumption.</p>
```

Comment:
I suggest adding here something along the lines of "and increasingly for creation of audiovisual media" (with link to the Media Production workshop report), to cover some of use cases for Web Codecs.

Reply:
Good point. I extended the sentence accordingly.